be/src/util/s3_util.cpp

```diff
@@ -135,7 +135,7 @@ bool S3ClientFactory::is_s3_conf_valid(const std::map<std::string, std::string>&
 }
 
 bool S3ClientFactory::is_s3_conf_valid(const S3Conf& s3_conf) {
-    return !s3_conf.ak.empty() && !s3_conf.sk.empty() && !s3_conf.endpoint.empty();
+    return !s3_conf.endpoint.empty();
```

Comment:
Why only endpoint is required?

Reply:
Because we must set it to aws config:
aws_config.endpointOverride = s3_conf.endpoint;